app/main/menu.ts

```diff
@@ -476,6 +491,21 @@ function getOtherTpl(props: MenuProps): Electron.MenuItemConstructorOptions[] {
 			accelerator: 'Ctrl+,',
 			click(_item, focusedWindow) {
 				if (focusedWindow) {
+					let numberServers = 0;
+					tabs.forEach(tab => {
+						if (tab.role === 'server') {
+							numberServers++;
+						}
+					});
+					if (numberServers === 1) {
+						const value = ConfigUtil.getConfigItem('showSidebar');
+						if (!value) {
+							const newValue = !value;
+							focusedWindow.webContents.send('toggle-sidebar', newValue);
+							ConfigUtil.setConfigItem('showSidebar', newValue);
+						}
+					}
+
```

Comment:
I think it'd probably be better to avoid duplicating this code, by writing a new function and calling it from both handlers.

Reply:
Avoided duplication of code by adding it in main.ts file.